generator/views/metric_definitions_view.py

```diff
@@ -283,7 +285,10 @@ def to_lookml(self, bq_client, v1_name: Optional[str]) -> Dict[str, Any]:
         return {"views": [view_defn]}
 
     def get_dimensions(
-        self, _bq_client=None, _table=None, _v1_name: Optional[str] = None
+        self,
+        _table=None,
+        _v1_name: Optional[str] = None,
+        dryrun=False,
```

Comment:
To match the convention of the other unused arguments should `dryrun` be prefixed with an underscore and have its default value be `None`?
```suggestion
        _dryrun=None,
```